server/data/DraftAdjudicationResult.ts

```diff
@@ -155,7 +155,7 @@ export enum WitnessCode {
 }
 
 export type HearingDetails = {
-  id: number
+  id?: number
   locationId: number
```

Comment:
oicHearingType field is now available on dev server and is mandatory so should really add under this PR, and default to GOV until other changes are made?